config/routes.rb

```diff
@@ -39,6 +39,7 @@
 
 
   namespace :api do
+    get 'logged_in' => 'api#logged_in'
```

Comment:
yeah, it is for me. It goes back a pretty long ways to HTML webpage dev days - imagine http://www.bob.com/this_is_awesome where the default link underlining is in play

![image](https://cloud.githubusercontent.com/assets/63355/14692352/2b0c0d9c-071c-11e6-93d4-04090eae873c.png)


vs

![image](https://cloud.githubusercontent.com/assets/63355/14692357/32cbb1e0-071c-11e6-9f1b-412d418b0f3b.png)